.taskcluster.yml

```diff
@@ -75,6 +75,7 @@ tasks:
           - sh
           - -lxce
           - "apk add --no-cache git img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing --quiet &&
+            apk add --no-cache musl-dev gcc &&
```

Comment:
If we have to install a compiler, we might as well switch to the standard python image instead of the one based on Alpine.
Like we did in https://github.com/mozilla/task-boot/commit/c1184c2d922061f34c36037f80584027e56b7b24 or in https://github.com/mozilla/task-boot/commit/c1d297adf3f01d4efba14514207ea2b429799f5f#diff-a728f7e52d751b98eafa856e45594533339b44f229d7b83f930df335391e7e15L105.

Reply:
Yep, let me try to use a `Python` image directly